### DISCUSSION TITLE
Some issues need to be closed

### DISCUSSION THREAD

**Discussion by SunsetTechuila** [nimsandu/spicetify-bloom#224] [General]
@nimsandu
https://github.com/nimsandu/spicetify-bloom/issues/114
https://github.com/nimsandu/spicetify-bloom/issues/130
https://github.com/nimsandu/spicetify-bloom/issues/131
https://github.com/nimsandu/spicetify-bloom/issues/149
https://github.com/nimsandu/spicetify-bloom/issues/174
https://github.com/nimsandu/spicetify-bloom/issues/198
https://github.com/nimsandu/spicetify-bloom/issues/207

**Comment by nimsandu** [comment]
ill check those out!
thanks

**Comment by nimsandu** [comment]
@SunsetTechuila done !

**Reply by SunsetTechuila** [reply]
#150 
#138
